### ISSUE TITLE
View list of products

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/openfoodnetwork#12245]
The Product images are not scaling correctly / staying square - on most screen views they are distorted

![Image](https://github.com/openfoodfoundation/openfoodnetwork/assets/2911176/fcf0bbda-8277-4c16-aa3c-51a2368a6224)

**Comment by JbPasquier** [comment]
Fixed.

![Capture d’écran 2024-03-07 à 00 28 29](https://github.com/openfoodfoundation/openfoodnetwork/assets/14278890/0b12ea4e-d5a4-4ea5-aee2-e3d8be885b4a)

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/14278890/3e28ae53-ea9d-48b8-b808-51c2a29ad97f)

**Comment by kirstenalarsen** [comment]
Thanks @JbPasquier that's much better! Is there a way to have BOTH the ratio locked and the actual squaring of the image?

**Comment by mariocarabotta** [comment]
> Thanks [@JbPasquier](https://github.com/JbPasquier) that's much better! Is there a way to have BOTH the ratio locked and the actual squaring of the image?

I'm not suggesting how it should be implemented, but by doing a quick search this is the kind of outcome for image cropping > https://jonathannicol.com/blog/2014/06/16/centre-crop-thumbnails-with-css/

**Comment by JbPasquier** [comment]
> Is there a way to have BOTH the ratio locked and the actual squaring of the image?

I can try some tricks, but it would be a better practice to handle this beforehand, and ensure a correct ratio when a producer update its picture.

> I'm not suggesting this post is how it should be implemented, but by doing a quick search this is the kind of outcome for image cropping > https://jonathannicol.com/blog/2014/06/16/centre-crop-thumbnails-with-css/

I just tried, does not work properly, because of the combination with the flex system I used for the mosaic,  
The main issue is that those squares have a dynamic height/width based on the ratio of the picture.  
In your link, they create a personalized class for portrait to address this issue, but here, I have no clue on the ratio of the image before it's on screen, and the logic approach of our framework is to avoid doing those manipulation on client side.

I'll try some other things and let you know. Let me know if you have any way to handle it sooner on the process.
